pkg/exthttp/hedging.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package exthttp
+
+import (
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/caio/go-tdigest"
+	"github.com/cristalhq/hedgedhttp"
+)
+
+type CustomBucketConfig struct {
+	HedgingConfig HedgingConfig `yaml:"hedging_config"`
```

Comment:
Could we update Thanos Store docs to show the fields, their purpose, the cost vs. benefit analysis (e.g. bigger `up_to` == more costs typically because providers charge per-request).

Reply:
I believe this is the correct doc to update (https://thanos.io/tip/components/store.md) ?

Reply:
Yup that is the doc